site/content/extend/plugins/developer-setup.md

```diff
@@ -18,7 +18,7 @@ For developing on Mattermost-managed plugins, each plugin's setup instructions c
 
 **Note:** Deploying with local mode will only work for plugins that have been updated with the functionality from the [Plugin Starter Template](https://github.com/mattermost/mattermost-plugin-starter-template).
 
-If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying your plugin. Edit your server configuration as follows:
+If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/manage/mmctl-command-line-tool.html) to streamline deploying your plugin. Edit your server configuration as follows:
```

Comment:
This update is partially correct. The full URL has been updated to include the target section.